### PR TITLE
Classification

### PR DESCRIPTION
**Pull request by tsundokum**

### CODE REVIEW

File: src/cljs/classification_checker/controls.cljs [hunk 1]
```diff
@@ -44,13 +71,13 @@
...
                              (reduce {} (.getKeys goog/object values))))))
-        (dispatcher/emit :email-received (user/user-info {:email email})))))
+        (dispatcher/emit :session-created (user/user-info {:email email})))))
 
   (ant/create-form (fn [_] (let [form (ant/get-form) submit-handler #(ant/validate-fields form submit-form-if-valid)]
                                  [:div {:style { :display "flex" :align-items "center" :justify-content "center" :height "100%"} }
                                   [ant/form {:layout "horizontal" :on-submit #(do (.preventDefault %) (submit-handler))}
                                    [ant/form-item {:label "Email"}
-                                    (ant/decorate-field form "email" {:rules [{:required true} {:type "email"}]} [ant/input])]
+                                    (ant/decorate-field form "email" {:initial-value "rar@gds.rr" :rules [{:required true} {:type "email"}]} [ant/input])] ;TODO
```
Comment (bavadim) [inline src/cljs/classification_checker/controls.cljs]: revert this

File: src/cljs/classification_checker/core.cljs [hunk 2]
```diff
@@ -42,4 +44,6 @@
     {:nav-handler (fn [path] (secretary/dispatch! path))
      :path-exists? (fn [path] (secretary/locate-route path))})
   (accountant/dispatch-current!)
-  (reagent/render [current-page] (.getElementById js/document "app")))
+  (reagent/render [current-page] (.getElementById js/document "app"))
+
+  (dispatcher/emit :session-created (user-info "test@test.com")))
```
Comment (bavadim) [inline src/cljs/classification_checker/core.cljs]: revert this

File: src/cljs/classification_checker/core.cljs [hunk 3]
```diff
@@ -1,28 +1,30 @@
...
             [classification_checker.store :refer [current-task]]
             [classification_checker.services :refer [create-session! upload! redirect!]]
             [classification_checker.dispatcher :as dispatcher]
             [reagent.session :as session]
             [reagent.core :as reagent]
             [secretary.core :as secretary :include-macros true]
             [accountant.core :as accountant]
-            [clojure.string :as string]))
+            [clojure.string :as string]
+            [classification_checker.user :refer [user-info]]))
```
Comment (bavadim) [inline src/cljs/classification_checker/core.cljs]: revert this

File: src/cljs/classification_checker/services.cljs [hunk 4]
```diff
@@ -18,23 +21,23 @@
   (let [packer :edn
         {:keys [chsk ch-recv send-fn state]} (sente/make-channel-socket! "/data" {:type :auto :packer packer})]
     (def chsk       chsk)
-    (def ch-items    ch-recv) ; ChannelSocket's receive channel
```
Comment (bavadim) [inline src/cljs/classification_checker/services.cljs]: move to let

Comment (tsundokum) [reply src/cljs/classification_checker/services.cljs]: it breaks, appears that `send-fn` must be defined in namespace scope

Comment (tsundokum) [reply src/cljs/classification_checker/services.cljs]: `(reset! upload_ chsk-send!)` -- that is the reason

**Pull request merged**